kubernetes/loculus/values.yaml

```diff
@@ -612,15 +624,21 @@ defaultOrganismConfig: &defaultOrganismConfig
       - name: gcaAccession
         displayName: GCA accession
         customDisplay:
-          type: link
+          type: linkWithMenu
```

Comment:
the `linkMenuItems` entry for the `gcaAccession`s is missing I think?

Reply:
cool, happy for you to push something to this branch or for us to do this downstream - whatever's easier!

Reply:
I will push - can you also make this a breaking change to remind us to make these config changes in PPX?